ntbs-service/Pages/Index.cshtml

```diff
@@ -5,244 +5,157 @@
-                    </a>
-                </nhs-table-item>
-                <nhs-table-item>
-                    @Html.DisplayFor(_ => item.FormattedCreationDate)
-                </nhs-table-item>
-                <nhs-table-item>
-                    @Html.DisplayFor(_ => item.TBServiceName)
-                </nhs-table-item>
-                <nhs-table-item>
-                    @Html.DisplayFor(_ => item.Episode.CaseManagerName)
-                </nhs-table-item>
-            </nhs-table-body-row>
-        }
-    </nhs-table-body>
-</nhs-table>
-
-<h2> Recent notifications </h2>
-<nhs-table class="notifications-table">
-    <nhs-table-head>
-        <nhs-table-item>
-            @Html.DisplayNameFor(model => model.RecentNotifications[0].NotificationId)
-        </nhs-table-item>
-        <nhs-table-item>
-            @Html.DisplayNameFor(model => model.RecentNotifications[0].FormattedSubmissionDate)
-        </nhs-table-item>
-        <nhs-table-item>
-            @Html.DisplayNameFor(model => model.RecentNotifications[0].TBServiceName)
-        </nhs-table-item>
-        <nhs-table-item>
-            @Html.DisplayNameFor(model => model.RecentNotifications[0].Episode.CaseManagerName)
-        </nhs-table-item>
-    </nhs-table-head>
-    <nhs-table-body>
-        @foreach (var item in Model.RecentNotifications) {
-            <nhs-table-body-row>
-                <nhs-table-item>
-                    <a href="@RouteHelper.GetNotificationPath(NotificationSubPaths.Overview, item.NotificationId)" class="govuk-link--no-visited-state">
-                        @Html.DisplayFor(modelItem => item.NotificationId)
-                    </a>
-                </nhs-table-item>
-                <nhs-table-item>
-                    @Html.DisplayFor(modelItem => item.FormattedSubmissionDate)
-                </nhs-table-item>
-                <nhs-table-item>
-                    @Html.DisplayFor(modelItem => item.TBServiceName)
-                </nhs-table-item>
-                <nhs-table-item>
-                    @Html.DisplayFor(modelItem => item.Episode.CaseManagerName)
-                </nhs-table-item>
-            </nhs-table-body-row>
-        }
-    </nhs-table-body>
-</nhs-table>
+                            </nhs-table-item>
+                            <nhs-table-item>
+                                @{var caseManagerAndTbService = String.Join(", ", item.CaseManagerFullName, item.TbServiceName);}
+                                @caseManagerAndTbService
+                            </nhs-table-item>
+                            <nhs-table-item>
+                                <form method="post" action="@RouteHelper.GetAlertPath(AlertSubPaths.Dismiss, @item.AlertId)" >
+                                    @Html.AntiForgeryToken()
+                                    <input hidden value="@item.TbServiceCode" name="hi"/>
+                                        <button class="nhsuk-link govuk-link--no-visited-state reset-button-styling">
```

Comment:
We'll want to give this a better name for screen readers. `aria-label="Dismiss alert"` , perhaps?